database/migrations/2025_05_05_000003_update_gamedata.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('GameData', function (Blueprint $table) {
+            $table->integer('times_beaten')->default(0)->after('players_hardcore');
+            $table->integer('times_beaten_hardcore')->default(0)->after('times_beaten');
+            $table->integer('median_time_to_beat')->nullable()->default(null)->after('times_beaten_hardcore');
+            $table->integer('median_time_to_beat_hardcore')->nullable()->default(null)->after('median_time_to_beat');
```

Comment:
These are the beaten aggregated metrics that would be displayed on the game page.

![image](https://github.com/user-attachments/assets/44b21994-dba7-42af-b556-fb7a25587f48)